openshift/tests-extension/pkg/helpers/cluster_extension.go

```diff
@@ -2,25 +2,27 @@ package helpers
 
 import (
 	"context"
+	"fmt"
+	"time"
```

Comment:
@kuiwang02 @tmshort 

It is an improvement:
- I just want to ensure that the tests are executed in a new namespace created for each test
- Also, I refac the helper since it will be used by other tests as well in a way that make easier 

Could you help me get this one merged?